test/box-luatest/gh_7897_sending_arbitrary_iproto_packets_test.lua

```diff
@@ -0,0 +1,448 @@
+local server = require('luatest.server')
```

Comment:
For features we do not use `gh_####` prefix in new test files.

Reply:
Any assertion needs to be backed up by some sort of reasoning, e.g., a developer guideline in this case.

AFAIS, there are feature test files with `gh_` prefix, for instance, bc053c55

Reply:
Here is the guideline: https://github.com/tarantool/tarantool/wiki/Code-review-procedure#testing. As for the example, it is also wrong then. For more feature tests you can open any test folder and look at tests not having `gh_####` or `gh-####` prefix, there are plenty.

Reply:
Thanks for pointing this out, fixed!